doc/userguide/src/CreatingTestData/AdvancedFeatures.rst

```diff
@@ -371,9 +371,9 @@ lower limit, upper limit and step.
 .. table:: For in range examples
    :class: example
 
-   ================  ===============  ===========  =========  ===========  ========  =======
-      Test Case          Action        Argument     Argument    Arg          Arg       Arg
-   ================  ===============  ===========  =========  ===========  ========  =======
+   ================  ===============  ===========  =========  ===========  ========  ========
+      Test Case          Action        Argument     Argument    Argument   Argument  Argument
+   ================  ===============  ===========  =========  ===========  ========  ========
```

Comment:
+1 for cleaning up example headers. @laurentbristiel is actually working to get all these examples converted to the plain text format. We are creating a bit more work for him with these new examples but I think it's still better to use the old style until his PR is merged.